updater/src/main/java/Main.java

```diff
@@ -0,0 +1,206 @@
+import static java.util.stream.Collectors.joining;
+import static okio.Okio.buffer;
+import static okio.Okio.sink;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import okhttp3.OkHttpClient;
+import retrofit2.Call;
+import retrofit2.Retrofit;
+import retrofit2.converter.moshi.MoshiConverterFactory;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public final class Main {
+  public static void main(String... args) throws IOException {
+    var repoDir = Paths.get(args[0]).toAbsolutePath().normalize();
+    System.out.println("Repository dir: " + repoDir);
+
+    var readmeFile = repoDir.resolve("README.md");
+    var caskDir = repoDir.resolve("Casks");
+    var workflowsDir = repoDir.resolve(".github/workflows");
+
+    Map<Integer, BundleSet> jdkBundles = loadJdks();
+    for (var entry : jdkBundles.entrySet()) {
+      int jdkVersion = entry.getKey();
+      var x86Bundle = entry.getValue().x86;
+      var armBundle = entry.getValue().arm;
+
+      var caskFile = caskDir.resolve("zulu-jdk" + jdkVersion + ".rb");
+      try (var w = buffer(sink(caskFile))) {
+        w.writeUtf8("cask 'zulu-jdk" + jdkVersion + "' do\n\n");
+
+        if (armBundle != null) {
+          w.writeUtf8("  if Hardware::CPU.intel?\n");
+        }
+
+        w.writeUtf8("    version '" + x86Bundle.version() + "'\n");
+        w.writeUtf8("    sha256 '" + x86Bundle.sha256_hash + "'\n\n");
+        w.writeUtf8("    url '" + x86Bundle.url + "',\n");
+        w.writeUtf8("        referer: 'https://www.azul.com/downloads/zulu-community/'\n\n");
+        w.writeUtf8("    depends_on macos: '>= :high_sierra'\n");
```

Comment:
May should be >= `mojave`?

https://www.azul.com/downloads/?version=java-7-lts&os=macos&architecture=x86-64-bit&package=jdk

Reply:
I would prefer to address this separately since it affects compatibility and I'm not well versed in how the supported versions vary.